### ISSUE TITLE
Consuming locally built h5web/lib fails to compile

### ISSUE THREAD

**Issue by PeterC-DLS** [silx-kit/h5web#1336]
### Describe the bug

I can build and publish h5web/lib to a local (verdaccio) registry. When I install this package as a dependency in another project, that build fails with import errors.

### To Reproduce

1. Clone h5web repo
2. Edit `packages/lib/package.json` and bump up version `0.0.2-pc`
3. Start a local registry (see, e.g., [verdaccio in a container](https://verdaccio.org/docs/docker#running-verdaccio-using-docker))
4. Follow steps in release workflow but use `--registry http://localhost:4873` on the publish step
5. Clone [vite app repo](https://github.com/PeterC-DLS/vite-h5weblib-app)
6. Install and build with
 ```
$ pnpm install --registry http://localhost:4873
$ pnpm build
> vite-react-app@0.0.0 build /scratch/tmp/src/vite-react-app
> tsc && vite build

src/App.tsx:1:31 - error TS2305: Module '"@h5web/lib"' has no exported member 'getMockDataArray'.

1 import { HeatmapVis, LineVis, getMockDataArray, getDomain } from '@h5web/lib';
                                ~~~~~~~~~~~~~~~~

src/App.tsx:1:49 - error TS2724: '"@h5web/lib"' has no exported member named 'getDomain'. Did you mean 'Domain'?

1 import { HeatmapVis, LineVis, getMockDataArray, getDomain } from '@h5web/lib';
                                                  ~~~~~~~~~


Found 2 errors in the same file, starting at: src/App.tsx:1

 ELIFECYCLE  Command failed with exit code 2.
 ```

### Expected behaviour

Build succeeds and then can be tested with `pnpm dev`.  Note changing the `@hweb/lib` version to `^6.0.0` brings in the publicly released version.

### Screenshots

None

### Context

- OS: RHEL 8
- Browser: n/a
- Version: n/a
- H5Web context: @h5web/lib

**Comment by axelboc** [comment]
In step 2, you publish version `0.0.2-pc`, but in the repo, the version installed is `0.0.2`. Isn't there a mismatch? Perhaps `0.0.2` is not found in the Verdaccio registry and therefore fetches the `0.0.2` version from NPM, which definitely doesn't include `getMockDataArray` for instance.

**Comment by PeterC-DLS** [comment]
Ah, thanks, I fixed it using `6.7.0`. It's a little confusing that the version in `packages/lib` is set at `0.0.1`.

**Comment by axelboc** [comment]
Yeah, the version gets bumped and committed on the root `package.json` of the monorepo, because that's easy to do with `pnpm version` (which forwards to `npm version` and creates a tag automatically). It's the CI that then takes care of applying that version number to the packages' `package.json` files just before publishing -- and those version bumps are not committed to avoid having an extra commit after the tagged commit. Sorry for the confusion :wink:
